lib/ansible/plugins/terminal/eos.py

```diff
@@ -49,6 +49,8 @@ class TerminalModule(TerminalBase):
         re.compile(br"% More than \d+ OSPF instance", re.I),
         re.compile(br"% Subnet [0-9a-f.:/]+ overlaps", re.I),
         re.compile(br"Maximum number of pending sessions has been reached"),
+        # match v4 & v6 prefix length errors
+        re.compile(br"% Prefix length must be less than (?:32|128)"),
```

Comment:
```suggestion
        re.compile(br"% Prefix length must be less than"),
```
We can almost certainly get away with being slightly less precise with this error.

Reply:
Agreed, committed.